lib/index.js

```diff
@@ -64,9 +100,9 @@ export default class Pageres {
 	 * @api public
 	 */
 
-	async run() {
+	async run(): Promise<stream$Readable[]> { // TODO: any way to define that it returns `this.items` and use its type?
```

Comment:
It can do that, but since you're building a library it's best to type as much of the exposed interface as possible. Check out this blog post I wrote: https://medium.com/@thejameskyle/flow-mapping-an-object-373d64c44592